Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -23,6 +23,15 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
+    double initialAhUsed_;
+    double chargeAh_;
+    double averageCurrent_;
+    double totalCurrent_ = 0;
```

Comment:
Typically values aren't assigned in the header file, if you need to assign a starting value, use the constructor!